### DISCUSSION TITLE
Do we extend or get new cert after expiration?

### DISCUSSION THREAD

**Discussion by teksal** [acmesh-official/acme.sh#5244] [Q&A]
Hello,
Sorry if this is a silly question, but it is the first time I make use of SSL certificates.
I successfully obtained and installed a certificate in DNS manual mode, which is going to expire in a week. 
So how do I renew it?
1.) Do I just re-run `acme.sh --renew ...` and magically extend the expiry date of existing certificate?
2.) Or, do I get a new certificate via `acme.sh --issue ...`, edit TXT record in DNS and then run `acme.sh --renew ...`?

**Comment by neilpang** [comment]
>> 1.) Do I just re-run acme.sh --renew ... and magically extend the expiry date of existing certificate?

**Reply by teksal** [reply]
Hi,
I've run `--renew` , got new certificates, `acme.sh --list` displays the new dates, updated the TXT record in DNS, copied the new certs to web server folder and restarted the server, but the client browser still shows the old dates. What am I missing? My cert is from ZeroSSL
